lib/recurly/resource.rb

```diff
@@ -15,7 +15,8 @@ def requires_client?
     end
 
     def ==(other_resource)
-      self.attributes == other_resource.attributes
+      other.is_a?(Recurly::Resource) &&
```

Comment:
The method parameter is `other_resource`

Reply:
Good catch!
Stupid error while renaming variables.
Thank you!